tree/ntuple/v7/src/RNTuple.cxx

```diff
@@ -42,7 +42,22 @@ void ROOT::Experimental::RNTuple::Streamer(TBuffer &buf)
       auto offCkData = offClassBuf + sizeof(UInt_t) + sizeof(Version_t);
       auto checksum = XXH3_64bits(buf.Buffer() + offCkData, lenCkData);
 
-      buf.ReadClassBuffer(RNTuple::Class(), this, classVersion, offClassBuf, bcnt);
+      buf >> fVersionEpoch;
+      buf >> fVersionMajor;
+      buf >> fVersionMinor;
+      buf >> fVersionPatch;
+      buf >> fSeekHeader;
+      buf >> fNBytesHeader;
+      buf >> fLenHeader;
+      buf >> fSeekFooter;
+      buf >> fNBytesFooter;
+      buf >> fLenFooter;
```

Comment:
Very nice! Let's add an additional check 
  - that `bcnt` is large enough to hold all the members that are deserialized
  - that the buffer actually large enought for `bcnt` bytes (this check should move up, before the checksum calculation)

Let's also discuss how we can unit test the streamer.